tests/data/sample_admission_requests/ad_request_replicaset_to_zero.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Not sure whether we have it, but if not, add a test for replica being set to something but 0 and checking that it fails

Reply:
added additional tests